docs/data/material/components/material-icons/synonyms.js

```diff
@@ -2948,6 +2948,7 @@ const synonyms = {
     'cancel close destination direction exit maps no pin place quit remove stop',
   Wysiwyg:
     'composer mode screen software system text view visibility website window',
+  X: 'twitter brand logo social',
```

Comment:
For some reason, it's not working:

<img src="https://github.com/mui/material-ui/assets/3165635/56f3915f-b34e-492d-b097-cbb4f9f38c0f" width="436">

https://deploy-preview-40193--material-ui.netlify.app/material-ui/material-icons/?query=twitter

I have added this problem into https://github.com/mui/material-ui/issues/38480.